adafruit_displayio_layout/widgets/cartesian.py

```diff
@@ -0,0 +1,514 @@
+                        self._axes_line_thickness,
+                        self._tick_line_height // 2,
+                        1,
+                        self._axesx_bitmap,
+                        1,
+                        self._screen_palette,
+                        True,
+                    )
+
+        # Y axes ticks
+        for i in range(10, 100, 10):
+            text_tick = str(round(self._yrange[0]) + round(i * self._normy))
+            text_dist = int(self._valuey * i)
+            if i in ticks:
+                shift_label_x = len(text_tick) * self._font_width
+                tick_text = bitmap_label.Label(
+                    self._font,
+                    color=self._font_color,
+                    text=text_tick,
+                    x=-shift_label_x
+                    - self._axes_line_thickness
+                    - self._tick_line_height
+                    - 2,
+                    y=0 + self.height - text_dist,
+                )
+                self.append(tick_text)
+                rectangle_helper(
+                    self._axesy_width
+                    - self._axes_line_thickness
+                    - self._tick_line_height
+                    - 1,
+                    text_dist,
+                    self._tick_line_thickness,
+                    self._tick_line_height,
+                    self._axesy_bitmap,
+                    1,
+                    self._screen_palette,
+                    True,
+                )
+
+            if self._subticks:
+                if i in subticks:
+                    rectangle_helper(
+                        self._axesy_width
+                        - self._axes_line_thickness
+                        - self._tick_line_height // 2
+                        - 1,
+                        text_dist,
+                        1,
+                        self._tick_line_height // 2,
+                        self._axesy_bitmap,
+                        1,
+                        self._screen_palette,
+                        True,
+                    )
+
+    def _draw_pointers(self, x: int, y: int) -> None:
+        self._pointer = vectorio.Circle(self._pointer_radius)
+        self._circle_palette = displayio.Palette(2)
+        self._circle_palette.make_transparent(0)
+        self._circle_palette[1] = self._pointer_color
+
+        self._pointer_vector_shape = vectorio.VectorShape(
+            shape=self._pointer,
+            pixel_shader=self._circle_palette,
+            x=x,
+            y=y,
+        )
+        self.append(self._pointer_vector_shape)
+
+    def update_pointer(self, x: int, y: int) -> None:
+        """updater_pointer function
+        helper function to update pointer in the plane
+        :param int x: ``x`` coordinate in the local plane
+        :param int y: ``y`` coordinate in the local plane
+        :return: None
+        rtype: None
+        """
+        local_x = int((x - self._xrange[0]) * self._factorx)
```

Comment:
I experimented a bit with the pointer position to try to get 0,0 matching the bottom left corner.
I updated this line like this: 
```
local_x = int((x - self._xrange[0]) * self._factorx) - 2
```
and it seems to be lining up correctly for me like this in a few quick tests. Do you think this would cause trouble in some situations @jposada202020? or would it be an okay fix to use?

Reply:
@FoamyGuy I think is perfect, doing the equalizer I realized that we need a `buffer` to correct all the float to int conversion, I think it is perfect, errors will be added at the end, kmatch uses a `nudge` variable to do this. Maybe following this line of thought instead of adding the `2` we could add the `nudge` correction, what do you think?

Reply:
@jposada202020 That sounds good to me

Reply:
@FoamyGuy Thanks, I add the ``nudge`` parameter